iconsMapper.ts

```diff
@@ -8,15 +23,66 @@ const mdiViewBox = {
 
 const remap = reMapIcons();
 
+/**
+ * Get the icon lookup from the icon string
```

Comment:
This is weirdly phrased, please rephrase.
Please explain at the parameter doc what's the expected input format